src/styles/stars.css

```diff
@@ -0,0 +1,26 @@
+/* #region Stars */
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  margin-right: 4px;
```

Comment:
please exclude last star of every row, there is no need for it to have some margin